test/admin/deploy.test.js

```diff
@@ -18,13 +18,6 @@ describe('Deploy', () => {
 
   afterEach(async() => th.cleanup());
 
-  describe('when requesting the deploy history', () => {
```

Comment:
i'm not sure how it was ever passing (we wouldn't have known since the suite's always been red anyway). it seems to rely on the assumption that the deploy history always has at least one entry after the app is created, which does not seem to be the case.